tests/odh/resources/mnist_hpo_raytune.ipynb

```
@@ -1,260 +1,265 @@
+        "num_gpus = \"has to be specified\""
+      ],
+      "execution_count": null,
+      "outputs": [],
+      "id": "30888aed"
+    },
+    {
+      "cell_type": "code",
+      "metadata": {},
+      "source": [
+        "auth = TokenAuthentication(\n",
+        "    token=kubernetes_user_bearer_token,\n",
```

Ruff (0.15.12)

[error] Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)